src/pages/SearchResultPage/SearchResultSidebar/Menu2Content.tsx

```diff
@@ -1,21 +1,108 @@
 import React from 'react';
 
+import { routes } from '../../routes';
+
 export const Menu2Content = {
   items: [
     {
-      name: 'Item 1',
-      link: '/item',
+      name: 'Data sets',
+      link: routes.datasets,
+      icon: (
+        <svg
+          width="16"
+          height="17"
+          viewBox="0 0 16 17"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            d="M1.77778 8.5C1.77778 8.76358 2.18756 9.22253 3.13778 9.67305C4.368 10.2558 6.11289 10.6053 8 10.6053C9.88711 10.6053 11.632 10.2558 12.8622 9.67305C13.8124 9.22253 14.2222 8.76358 14.2222 8.5V6.67179C12.7556 7.53074 10.5129 8.07895 8 8.07895C5.48711 8.07895 3.24444 7.5299 1.77778 6.67179V8.5ZM14.2222 10.8823C12.7556 11.7413 10.5129 12.2895 8 12.2895C5.48711 12.2895 3.24444 11.7404 1.77778 10.8823V12.7105C1.77778 12.9741 2.18756 13.4331 3.13778 13.8836C4.368 14.4663 6.11289 14.8158 8 14.8158C9.88711 14.8158 11.632 14.4663 12.8622 13.8836C13.8124 13.4331 14.2222 12.9741 14.2222 12.7105V10.8823ZM0 12.7105V4.28947C0 2.19684 3.58222 0.5 8 0.5C12.4178 0.5 16 2.19684 16 4.28947V12.7105C16 14.8032 12.4178 16.5 8 16.5C3.58222 16.5 0 14.8032 0 12.7105ZM8 6.39474C9.88711 6.39474 11.632 6.04526 12.8622 5.46253C13.8124 5.012 14.2222 4.55305 14.2222 4.28947C14.2222 4.02589 13.8124 3.56695 12.8622 3.11642C11.632 2.53368 9.88711 2.18421 8 2.18421C6.11289 2.18421 4.368 2.53368 3.13778 3.11642C2.18756 3.56695 1.77778 4.02589 1.77778 4.28947C1.77778 4.55305 2.18756 5.012 3.13778 5.46253C4.368 6.04526 6.11289 6.39474 8 6.39474Z"
+            fill="#D3E9F5"
+          />
+        </svg>
```

Comment:
I would recommend to keep svg files in resource folder. Don't embed them into code

Reply:
Another one reason  - it is hard to read code, it is too verbose